--- setup/etc/yum.repos.d/commercial-addons.repo    2019-11-12 17:18:22.272194006 -0500
+++ setup/etc/yum.repos.d/commercial-addons.repo.new    2019-11-12 17:13:39.422156785 -0500
@@ -1,16 +1,10 @@
 [commercial-addons]
 name=Commercial-Addons RPM Repository for Issabel
-mirrorlist=http://mirror.issabel.org/?release=2.5&arch=$basearch&repo=commercial_addons
+mirrorlist=http://mirror.issabel.org/?release=4&arch=$basearch&repo=commercial_addons
 gpgcheck=1
 enabled=1
 gpgkey=http://repo.issabel.org/issabel/RPM-GPG-KEY-Issabel
 
-[LowayResearch]
-name=Loway Research Yum Repository
-baseurl=http://yum.loway.ch/RPMS
-gpgcheck=0
-enabled=0
-
 [iperfex]
 name=IPERFEX RPMs repository
 baseurl=http://packages.iperfex.com/centos/$releasever/$basearch/

